### PR TITLE
Shaheer/feature/90115/update jurisdiction

### PR DESCRIPTION
**Pull request by shaheer-deriv**
## Changes:

Please provide a summary of the change.

### Screenshots:

Please provide some screenshots of the change.

### CODE REVIEW

File: packages/api/src/APIProvider.tsx [hunk 1]
```diff
@@ -1,13 +1,14 @@
 import React, { PropsWithChildren } from 'react';
 import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
-import { ReactQueryDevtools } from '@tanstack/react-query-devtools';
+// import { ReactQueryDevtools } from '@tanstack/react-query-devtools';
```
Comment (suisin-deriv) [inline packages/api/src/APIProvider.tsx:3]: can we remove this commented code

File: .circleci/config.yml [hunk 2]
```diff
@@ -179,6 +179,7 @@ jobs:
   build:
     docker:
       - image: cimg/node:16.16.0
+    resource_class: xlarge
     steps:
```
Comment (hirad-deriv) [inline .circleci/config.yml:183]: These changes are unrelated to your PR

**Review by hirad-deriv — COMMENTED** [review]
Your feature branch needs to get updated. 153 file changes are being shown

File: packages/api/src/APIProvider.tsx (hunk 1 above)
Comment (shaheer-deriv) [reply packages/api/src/APIProvider.tsx:3]: @suisin-deriv i think this was intentional. please see this [link](https://deriv-group.slack.com/archives/C3AHMSM3P/p1685000609101639)

File: .circleci/config.yml (hunk 2 above)
Comment (shaheer-deriv) [reply .circleci/config.yml:183]: @hirad-deriv the feature branch is having conflicts with master branch, and therefore couldnt merge anything to it. thats why i created this branch to resolve conflict and merge it to feature branch.
Probably these unrelated changes happened in master branch after the conflict.

Comment (amina-deriv) [reply .circleci/config.yml:183]: @shaheer-deriv I think we can merge this PR to feature branch to resolve conflicts and review the main PR. It is hard to track your changes in this PR.

Comment (shaheer-deriv) [reply .circleci/config.yml:183]: @hirad-deriv if required, we can merge this PR without reviewing, and review the feature branch PR https://github.com/binary-com/deriv-app/pull/8093 
so that we don't see unrelated changes in the PR.

Comment (shaheer-deriv) [reply .circleci/config.yml:183]: @amina-deriv sure.. will merge this and lets review the feature branch PR.

Comment (shaheer-deriv) [reply .circleci/config.yml:183]: Feature branch PR: https://github.com/binary-com/deriv-app/pull/8093
